### PR TITLE
Closes Issue 106: Frontend: Notification Page created

### PR DESCRIPTION
**Pull request by Aniket1103**
## Related Issue

- Info about Issue or bug
Created Notification Page as per the Figma design 

Closes: #106

#### Describe the changes you've made

A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

Made some necessary changes and enhancements in Navbar.js, searchbar.js.
created NotificationPage.js, NotificationComp.js, sideBar.js, and DummyData.js and kept them in `src-->Component-->NotificationPage` directory.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.

## Screenshots

Screenshot (132)

Screenshot (133)

### CODE REVIEW

**Comment by Aniket1103** [comment]
@ir2010 and @purnima143  kindly review the changes I have made.

File: client/src/Component/NotificationPage/NotificationPage.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    else{
+        time = "Evening";
+    }
+    return (time);
+}
+
+function NotificationPage(){
+    return (
+        <div>
+            <Navbar></Navbar>
```
Comment (codewithvk) [inline client/src/Component/NotificationPage/NotificationPage.js:28]: Write like this `<Navbar />`

Comment (Aniket1103) [reply client/src/Component/NotificationPage/NotificationPage.js]: Done! @codewithvk

**Comment by Aniket1103** [comment]
@purnima143 plz review the PR and let me know if any more changes are to be made.

File: client/src/App.js [hunk 2]
```diff
@@ -5,22 +5,27 @@ import Feed from "./components/feed";
...
-      </BrowserRouter>
-    </>
+
+  <BrowserRouter>
+      <Switch>
+        <Route path="/" exact component={Homepage} />
+        <Route path="/signup"  component={Signup} />
+        <Route path="/signin"  component={Signin} />
+        <Route path="/feed"  component={Feed} />
+        <Route path="/notification" component={NotificationPage} />
```
Comment (purnima143) [inline client/src/App.js:21]: Don't modify any other lines...or copy-paste the same thing...If you're adding a notification page line add that particular line only. I hope you're understanding what I'm trying to tell you. These small things help you in future😊

**Comment by purnima143** [comment]
@Aniket1103  good work😊

File: client/src/App.js (hunk 2 above)
Comment (Aniket1103) [reply client/src/App.js:21]: Yea got it,
surely I will keep that in mind from next time👍
Thank you!

**Comment by Aniket1103** [comment]
Thank you @purnima143,
and plz do add `gssoc` and `level` labels.

**Comment by purnima143** [comment]
> Thank you @purnima143,
> and plz do add `gssoc` and `level` labels.

Done

**Pull request merged**